compiler-memberinjector/src/main/java/toothpick/compiler/memberinjector/generators/MemberInjectorGenerator.kt

```diff
@@ -161,4 +162,4 @@ internal class MemberInjectorGenerator(
             )
         }
     }
-}
+}
```

Comment:
You should run `./gradlew spotlessApply` to fix formatting and pass the lint checks :)